tasks/kmt.py

```diff
@@ -660,15 +862,22 @@ def test(
     domains = filter_target_domains(vms, infra)
     used_archs = get_archs_in_domains(domains)
 
+    if len(domains) == 0:
+        raise Exit(f"no vms found from list {vms}. Run `inv -e kmt.status` to see all VMs in current stack")
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 nitpick:** ‏Could be replaced by asserts